app/models/application_record.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Do we really need this file?

Reply:
Yes

Reply:
We need this file so as to have an ability to create modules and inherit them easily. We think wi may need this later.

https://redpanthers.co/application_record-rb-available-since-rails-5/

Reply:
Nice!